README.md

```diff
@@ -1 +1,146 @@
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+
+
+# 📖 [Today's To Do List] <a name="about-project"></a>
+
+A simple Todo List app.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+1. Linters
+2. Webpack
+3. HTML5
+3. JS (ES6 synthax)
+4. CSS3
+
+### Key Features <a name="key-features"></a>
+
+1. Mark tasks as complete: Add functionality that allows users to mark tasks as completed. This can be done by adding a checkbox next to each task and updating its status accordingly. 
+
+2. Delete Tasks: Allow users to delete tasks from the list after they are completed or no longer relevant. You can add a trash can icon next to each task to allow deletion. 
+
+3. Task sorting: Add the ability to sort tasks by different criteria, such as alphabetical order, date added, completion status, etc. It will help users organize their to-do list more conveniently. 
+
+4. Filter tasks: Allow users to filter tasks based on different criteria, such as showing only completed tasks, unfinished tasks, priority tasks, etc.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+### Prerequisites
+
+In order to run this project you need:
+
+install node and vscode
+linters configuration
+webpack configuration
+
+### setup
+<p>Clone this repository to your desired folder</p>
+<a href ="git@github.com:GhostEsso/toDoList.git"></a>
+
+### Install
+
+Install this project with: -Git clone on your cd Desktop using my repository link above
+
+### Usage
+
+-To run the project, copy its path and run it on your online server.
+
+
+
+### Run tests
+
+To run tests, run the following command:
+
+npx hint .
+npx stylelint "*/.{css,scss}"
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Author <a name="authors"></a>
+
+
+👤 **Author**
+
+- GitHub: [@GhostEsso](https://github.com/GhostEsso)
+- LinkedIn: [@essohanam](https://www.linkedin.com/in/essohanam-tambana-62aa081a1/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features">
+
+- Add more style with CSS;
+- Create contact form.
+
+
+
+</a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+Feel free to check the [issues page].
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project you can contact me via my contact address and rate it.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank all the microverse team and my coding partners.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [<a href = "MIT.md">MIT</a>](./MIT.md) licensed.
```

Comment:
- [ ] Please always make sure that your `README` don't have any broken link, As I can see that you didn't replace the `MIT.md` with `LICENSE` file name because of that whenever any person clicks on your `license` to see it give **404 page** error. Because your license file name is `LICENSE` file so please use it in the place of `MIT.md`. Kindly look at the below suggestion.

```suggestion
This project is [MIT](./LICENSE) licensed.
```

Reply:
@skyv26 Please, I wanted to know if I can modify the messages of my commits, is it possible?

Reply:
@GhostEsso It is ok, what you did but make sure everything is for the next time. It will help you to be a better developer with professionalism 👓